diff.py

```diff
@@ -281,9 +282,8 @@ def clean_hsp(hsp):
     sys.exit(0)
 
 # TODO:  refactor and put "diff_view" check in an if/else; get rid of exit(0)
-print(
-    f"Change report for {display_name} from {sorted_hsps[0]['captured_date']} to {sorted_hsps[-1]['captured_date']}\n\n"
-)
+if not args.from_date and args.to_date:
```

Comment:
this line needs to have a 'not' on each side like so:
```suggestion
if not args.from_date and not args.to_date:
```